packages/engine/lib/orchestrator/src/manifest.rs

```diff
@@ -88,26 +120,51 @@ impl Manifest {
         }
     }
 
+    /// Reads the JSON content from the file at the provided `path` describing the
+    /// [`Globals`](hash_engine::config::Globals).
```

Comment:
```suggestion
    /// Reads the content from the file at the provided `path` describing the
    /// [`Globals`](hash_engine::config::Globals).
```

The function doesn't know that it's JSON, and doesn't perform any validation or serialization of it